src/inmanta/server/services/metricservice.py

```diff
@@ -34,9 +34,10 @@
 class MetricsService(protocol.ServerSlice):
     """Slice managing metrics collector"""
 
-    def __init__(self) -> None:
+    def __init__(self, extra_tags: dict[str, str] = {"component": "server"}) -> None:
         super().__init__(SLICE_METRICS)
         self._influx_db_reporter: Optional[InfluxReporter] = None
+        self._extra_tags = extra_tags
```

Comment:
Or we can also make it a mapping instead of a dict.
```suggestion
        self._extra_tags = dict(extra_tags)
```